.github/docs/CHANGELOG.md

```diff
@@ -1,23 +1,51 @@
 # Changelog
+
+### v0.14.0
+
+* Tweaked loading messages
+
+* Removed unused `env_effects.glsl` file
+
+* Updated authors
+
+* Credits loading message replaced by a message that links to the website's credits page
+
+* Added a new preset toggle to control whether or not the underwater distortion effect is enabled
+  
+  * This toggle is disabled on the `Lowest` quality setting
```

Comment:
I've enabled it even on lowest settings cuz I can't notice any difference switching animations on/off even on lowes settings and 225x400 resolution (less pixels processing with the same vertexs count)

Reply:
Ok, I'll update the changelog to reflect this